### ISSUE TITLE
PMError overwrites global Error object

### ISSUE THREAD

**Issue by ragulka** [paymill/paymill-js#18]
Whenever I load the paymill-js module in npm, it overwrites the original javascript global `Error` object. Every single generic error now has `PMError` as their name. I believe the following line is wrong: https://github.com/paymill/paymill-js/blob/master/paymill.node.js#L276
```javascript
PMError.prototype = Error.prototype;
```
should be
```javascript
PMError.prototype.__proto__ = Error.prototype;
```
or
```javascript
PMError.prototype = new Error();
```

**Comment by bybrunobarros** [comment]
+1 quite annoying.

**Comment by stoilkov** [comment]
Thanks for the feedback, it's fixed with 2.0.3 and up in npm.
